### PR TITLE
Add ADR for: Internal identifiers SHOULD NOT be prefixed

### PR DESCRIPTION
**Pull request by nsheff**
Resolves half of #37

### CODE REVIEW

**Comment by nsheff** [comment]
1. "external" can mean 2 things: A) content provided to us; B) the way our identifiers are used externally. Should be clarified.
2. We need more explicit direction on how/when content provided to us should be prefixed.

File: docs/decision_record.md [hunk 1]
```diff
@@ -9,6 +9,31 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 [TOC]
 
 
+## 2023-XX-XX - Internal identifiers SHOULD NOT be prefixed
```
Comment (tcezard) [inline docs/decision_record.md:12]: Should update the date

**Review by tcezard — APPROVED** [review]

**Review by sveinugu — CHANGES_REQUESTED** [review]
I am happy with the pet of the ADR that refer to internal identifiers, however I think it is premature to approve the part on external without knowing the conclusion of a) whether refget digests must be prefixed or not, and b) any top-level decision on GA4GH CURIE namespacing. 

If for instance it is decided to use the same namespace for seqcol and refget identifiers, I would argue
that the refget identifiers should NOT be prefixed inside a seqcol (as the namespace would is the default one). 

In addition one could also add an explicit requirement that values that are defined as CURIEs (in the schema?) from other namespace MUST include the namespace prefix.

**Comment by nsheff** [comment]
decision: sequences *should not* be prefixed with `ga4gh:`, but they *should* be prefixed with `SQ.`

**Comment by nsheff** [comment]
Ok, I added this statement:

> More specifically, for refget, there are two types of prefix: the namespace prefix (`ga4gh:`) and type type prefix (`SQ.`). Right now, the refget server requires you to have the type prefix to request a lookup; the refget protocol declares that this type prefix is *part of the identifier*. However, the `ga4gh:` prefix is more of a namespace prefix and is *not* required, and therefore not considered part of the identifier. Therefore, the seqcol `sequence` values would *include* the `SQ.` but not the `ga4gh:`.

Is that correct?

**Pull request merged**